lab-justin/lib/client-pool.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const EventEmitter = require('events').EventEmitter;
+var uuid = require('node-uuid');
```

Comment:
Are you talking about uuid? - i see it in package.json "dependencies" ( see below )
{
  "name": "lab-justin",
  "version": "1.0.0",
  "description": "",
  "main": "server.js",
  "dependencies": {
    "node-uuid": "^1.4.7"
  },
  "devDependencies": {},
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "start": "node server.js"
  },
  "author": "justin",
  "license": "ISC"
}